org.mwc.debrief.track_shift/src/org/mwc/debrief/track_shift/TrackShiftActivator.java

```diff
@@ -61,6 +62,7 @@ public TrackShiftActivator() {
 	 */
 	@Override
 	public void start(final BundleContext context) throws Exception {
+		System.out.println("Started track shift");

```

Comment:
@ayeshare - we don't need these debug lines, do we?

Reply:
@IanMayo removed the debug lines